src/utils/canvas.ts

```diff
@@ -21,7 +21,13 @@ export async function drawBlobOnCanvas(
   }
 }
 
-export async function canvasToBlob(canvas: HTMLCanvasElement, mimeFormat: MimeType): Promise<Blob> {
+export async function canvasToBlob(
+  canvas: HTMLCanvasElement,
+  mimeFormat: MimeType | FormatJpegOrPng,
+): Promise<Blob> {
+  if (mimeFormat === MimeTypes.JPEG_OR_PNG) {
```

Comment:
I don't understand this. Why was it added to supported types if it is not supported? wouldn't it make more sense to 
- use correct MimeType instead of FormatJpegOrPng
- use mimeFormat: MimeType|string
- use canvas.toBlob instead of canvasToBlob wrapper

I can'

Reply:
I agree it was a mess. I now went with the second option, _use mimeFormat: MimeType|string_ 

> use canvas.toBlob instead of canvasToBlob wrapper

can be used anyway already, but it's annoying because you have to pass the callback function to it to use the generated blob ([docs](https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toBlob)).
With `canvasToBlob()` you can simply do  `const blob = await canvasToBlob(...)`

And I guess we don't have to worry about the unsupported formats. From the docs:
> The default type is `image/png`; that type is also used if the given type isn't supported.